app.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
+
+(function(){
+  angular
+  .module("todo", [])
+  .controller("todos_controller", ControllerFunction);
```

Comment:
I would expect this to throw an error since there is no `ControllerFunction` defined:

```
function ControllerFunction() { 
}
```